_lib/wordpress_event_processor.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+
+import sys
+import json
+import os.path
+import requests
+import dateutil.parser
+
+
+def posts_at_url(url):
+    """
+    Get WordPress posts as JSON from the given URL
```

Comment:
They're Python docstrings. They create API documentation that can be viewed with `pydoc` at the command line.